packages/cli/src/commands/apps/stacks/index.ts

```diff
@@ -0,0 +1,35 @@
+import {ux} from '@oclif/core'
+import {Command, flags} from '@heroku-cli/command'
+import * as _ from 'lodash'
+import color from '@heroku-cli/color'
+import {Stacks} from '../../../lib/types/stacks'
+import {App} from '../../../lib/types/app'
+
+export default class StacksIndex extends Command {
+  static description = 'show the list of available stacks'
+  static topic = 'apps'
+  static aliases = ['stack']
+
+  static flags = {
+    app: flags.app({required: true}),
+  }
+
+  async run() {
+    const {flags} = await this.parse(StacksIndex)
+
+    const {body: app} = await this.heroku.get<App>(`/apps/${flags.app}`)
+    let {body: stacks} = await this.heroku.get<Stacks>('/stacks')
+
+    stacks = _.sortBy(stacks, 'name')
+    ux.styledHeader(`${color.app(app.name)} Available Stacks`)
+    for (const stack of stacks) {
+      if (stack.name === app.stack.name) {
+        ux.log(color.green('* ' + stack.name))
```

Comment:
What was the reason for removing the `map` function? This will change output and I'm not sure of the consequences.

Reply:
I was confused why that function was there to begin with since it's not iterating over anything but a string. I confirmed the output working manually and within the tests. Am I missing something here with regards to the `map` function?

Reply:
The function is poorly named, but it essentially converts "cedar" to "cedar-10" or returns the string. Removing it will change output, which we should avoid.